.github/workflows/release.yml

```diff
@@ -0,0 +1,57 @@
+name: Release
+
+on:
+  push:
+    tags:
+      - "*"
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      packages: write
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Install Rust
+        uses: dtolnay/rust-toolchain@stable
+        with:
+          targets: aarch64-linux-android armv7-linux-androideabi i686-linux-android x86_64-linux-android
+
```

Comment:
Could you add these targets to the instructions in `java/README.md`, specifically the `rustup` command?

Reply:
Good idea, do you mean to add a step-by-step on how to locally cross-compile the library?

Reply:
When I tried running this locally, it failed because I didn’t have these targets set up, and I had to debug it myself. It would be helpful to include a command for adding these targets as a perquisite, something like:
```sh
rustup target add \
    aarch64-linux-android \
    armv7-linux-androideabi \
    ...
```

Reply:
Also, let’s include a note about the supported `ndkVersion`, because running this with a different version also causes it to completely fail locally for me (I had version `28.0.12674087`. This is why I suggested https://github.com/ARK-Builders/ark-core/pull/94#discussion_r1885367339)

with an annoying error as well (I think it said something like `ndk` not installed)